dev/com.ibm.ws.jaxb_fat/publish/servers/jaxb_internal_optional_feature_toleration_fat/server.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+    <featureManager>
+        <feature>jaspic-1.1</feature>	<!-- jpaContainer-2.1 PRIVATE feature contains jaxb-2.2 feature that we target to test it's jaxb-2.3 toleration -->
```

Comment:
the comment doesn't match the config here. We're enabling `jaspic-1.1` but the comment talks about `jpaContainer-2.1`

Reply:
corrected